完成style插件

Chikit-L opened the pull request.

Chikit-L committed:
完成Style

Chikit-L committed:
Merge branch 'zotero-chinese:main' into main

Chikit-L committed:
Delete src/user-guide/plugins/style.md

Chikit-L committed:
Update Zotero style.md

northword committed:
fix lint

northword commented on src/user-guide/plugins/style.md:
```diff
@@ -1,5 +1,684 @@
+如果你想设置标签，你可以在`标签`上右键，然后点击**列设置**，这是一切设置的起点，你在不同列名上右键，对应的列设置功能不同。
+
+![列设置](../../assets/image-zoterostyle-右键菜单.png){width=50%}
+
```
这里的图似乎与下面“列设置的设置过程”之需要一张即可。

northword committed:
暂存更改

- 压缩此pr新增的图片
- 修改部分内容
- 新增两个badge：`_Z6_`和 `_Z7_` ，将渲染为 “仅支持 Zotero 6” 和 “仅支持 Zotero 7” 以更好描述插件的兼容情况
- 移除部分图片：部分内容文字已足够清晰
- 替换部分图片：部分图片仓库已有，复用以减少文件冗余

northword commented on src/user-guide/plugins/style.md:
```diff
@@ -1,5 +1,690 @@
+
+您可以通过编辑`列设置`中的`Map`字段自定义标签的显示，使用字符串或 regex 将标签替换为您所需的格式。有关 regex 语法，请参阅 [链接](https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Guide/Regular_Expressions) 或其他教程。
+
+大多数标记由两部分组成：`field`和`value`，中间用空格隔开。有些标记只有字段。
```
这里是 = 分隔嘛？然后什么时候是只有字段呢？类似“SCI=”这种？这种我觉得也许写成值为空更好理解一些？

northword committed:
暂存：阅读至标注

northword committed:
暂存：前一次提交把视图组移动错位置了

northword committed:
压缩图片

northword committed:
tweak
